mix.exs

```diff
@@ -2,21 +2,27 @@ defmodule ExDoc.Mixfile do
   use Mix.Project
 
   def project do
-    [app:        :ex_doc,
```

Comment:
Let's please keep this syntax as it is the one used in all `elixir-lang` projects.